build.gradle

```diff
@@ -48,10 +48,15 @@ dependencies {
   // CSV Stuff
   compile group: 'org.apache.commons', name: 'commons-csv', version: '1.5'
   compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-csv', version: '2.8.8'
-  compile group: 'org.yaml', name: 'snakeyaml', version: '1.19'
+  compile group: 'org.yaml', name: 'snakeyaml', version: '1.25'
   compile group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
   compile group: 'org.apache.commons', name: 'commons-text', version: '1.2'
 
+  compile group: 'org.opencds.cqf', name: 'cql-engine', version: '1.2.20'
+  compile group: 'info.cqframework', name: 'cql', version: '1.2.20'
+  compile group: 'info.cqframework', name: 'model', version: '1.2.20'
+  compile group: 'info.cqframework', name: 'cql-to-elm', version: '1.2.20'
```

Comment:
These CQL library versions are out of date by a year.... update to newer version?

Reply:
Will do, thanks Chris!

Reply:
We have had some packaging issues with the 1.3.9 release, can you use the 1.3.10-SNAPSHOT?

Reply:
Hi Bryn. Sorry I just saw your comment. Using 1.3.10-SNAPSHOT appears to work, but we get a ton of DEBUG messages in the output.

Reply:
Try to override the log4j settings.

Reply:
For example, maybe these suggestions might work: https://stackoverflow.com/questions/4258849/override-log4j-properties

Reply:
I went ahead overriding the logging with log4j.xml